src/Multics.xml

```diff
@@ -27,6 +27,12 @@
          successor in interest by change in name only to Honeywell Bull Inc. and Honeywell Information Systems
          Inc.</p>
       </optional>
+	  <optional>
+	  .
+	  </optional>
+	  <optional>
+	  -----------------------------------------------------------
```

Comment:
I'd rather not be picky about the number of dashes in this ASCII horizontal rule.  Can we use:

```xml
<alt match="-*"><p>-----------------------------------------------------------</p></alt>
```

or similar?

Reply:
I added an alt tag similar, but without the paragraph tag in an updated PR